### ISSUE TITLE
Get the CI on v1.8 green

### ISSUE THREAD

**Issue by sebathomas** [magma/magma#14078]
At the moment, the CI is red on v1.8. See https://github.com/magma/magma/pull/13997#issuecomment-1259054348 for known issues but there might be more problems.

Backport the necessary commits (or create a new PR if it's not easy to backport).

Optional: Enable weekly CI run if it is easy. Discuss if we want to have it.

- [ ] Run the CI on v1.8. Make sure the CI is actually running and not skipped because of path filters.
- [ ] Get all the jobs running on a pull request on v1.8 clean (excluding flakiness and non-required stuff like coverage that is also broken on master).
- [ ] Discuss how to handle this in the future. Create a follow-up ticket, have a discussion in devops meeting...
- [ ] Prepare internal review meeting

**Comment by nstng** [comment]
```
Hi, the 1.8 CI is broken because libsystemd0 is included in the 1.8 artifactory in version 245.4-4ubuntu3.17 . This requires the dependency libsystemd-dev in 245.4-4ubuntu3.17 which is not in our artifactory and got removed from the ubuntu repos (in favor of 3.18).
We suggest to remove libsystemd0 from the 1.8 artifactory because it is not on focal-ci and it is not known why it even is on 1.8 (it is on 1.7 and might be copied wrongly to 1.8).
If there are any objections, then please speak up. If there are
[libsystemd-dev.zip](https://github.com/magma/magma/files/9918075/libsystemd-dev.zip)
 no objections then we will remove the artifact from 1.8 test and production next week
```
https://magmacore.slack.com/archives/C0301NRRL7K/p1666949274562359

@MoritzThomasHuebner deleted libraries from 1.8 debian-test and production. Please test if workflows are working now. Files are attached if they need to be re-added.

**Comment by Neudrino** [comment]
Resolved by #14108
